src/app/components/modal/modal.component.html

```diff
@@ -42,7 +42,7 @@
   </div>
 
   <div class="modal_footer">
-    <div class="button button_plain" (click)="close();$event.stopPropagation();">cancel</div>
-    <div class="button" (click)="loadDataSet();$event.stopPropagation();">Launch</div>
+    <div class="button button_plain" (click)="close();$event.stopPropagation();">Cancel</div>
+    <div class="button disabled" (click)="loadDataSet();$event.stopPropagation();">Launch</div>
```

Comment:
@cholson how'd you want the `disabled` class applied for the Launch button?

Reply:
from charles… "Make Launch unclickable (grayed out) if your dataset selected and a file is not uploaded and validated"